packages/salesforcedx-vscode-core/src/commands/forceSourceRetrieveSourcePath.ts

```diff
@@ -119,25 +119,6 @@ export const forceSourceRetrieveSourcePaths = async (
     uris.push(sourceUri);
   }
 
-  const messages: ConflictDetectionMessages = {
-    warningMessageKey: 'conflict_detect_conflicts_during_retrieve',
```

Comment:
I'm still seeing this message in tests + i18n; was it not needed, or moved around?

Reply:
Good eye.  It is not used at all.  Two routes: remove the tests and its entry in the i18 file, or leave this, since it will be needed again someday when conflict detection is enabled for retrieve.  Thoughts?